projects/storefrontlib/src/lib/my-account/close-account/components/close-account-modal/close-account-modal.component.ts

```diff
@@ -0,0 +1,61 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  RoutingService,
+  UserService,
+  UserToken,
+} from '@spartacus/core';
+
+import { Observable, Subscription } from 'rxjs';
+
+import i18next from 'i18next';
+
+@Component({
+  selector: 'cx-close-modal-account',
+  templateUrl: './close-account-modal.component.html',
+  styleUrls: ['./close-account-modal.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CloseAccountModalComponent implements OnInit {
+  private subscription = new Subscription();
+  userToken$: Observable<UserToken>;
+
+  constructor(
+    private activeModal: NgbActiveModal,
+    private userService: UserService,
+    private globalMessageService: GlobalMessageService,
+    private routingService: RoutingService
+  ) {}
+
+  ngOnInit() {
+    this.userService.resetRemoveUserProcessState();
+    this.subscription.add(
```

Comment:
you should unsubscribe. Looking at dev tools, I see that it is always there staying true.